src/components/accessories/patientSummary/patientSummaryByDate/PatientSummaryByDate.tsx

```diff
@@ -16,24 +16,21 @@ const label = {
 };
 
 const PatientSummaryByDate: FunctionComponent<TProps> = ({
-  hasSucceeded,
   loadSummaryData,
   isLoading,
 }) => {
-  const [summaryData, setSummaryData] = useState(Array<SummaryData>());
-  const summary = useSelector<IState, ISummaryState>((state) => state.summary);
+  const summaryData = useSelector<IState, Array<SummaryDataType>>((state) =>
+    state.summary.summaryData.data
+      ? state.summary.summaryData.data
+      : new Array<SummaryDataType>()
```

Comment:
There's no need to do that. You can declare a value for `summary` in the redux initial state. Also, there's some naming confusion here. The structure should be something like `state.summary.data` and the variable inside the component `PatientSummaryByDate` could simply be declared under the identifier `summary`.

Reply:
Ok, I improve it.

Reply:
I remove that variable declaration there.